process/outportBlockConverter.go

```diff
@@ -184,6 +760,24 @@ func (o *outportBlockConverter) castBigInt(i *big.Int) ([]byte, error) {
 	return buf, err
 }
 
+func copyMiniBlocks(sourceMiniBlocks []*block.MiniBlock) []*hyperOutportBlocks.MiniBlock {
+	var destMiniBlocks []*hyperOutportBlocks.MiniBlock
+	if sourceMiniBlocks != nil {
```

Comment:
we don't need this nil check, ranging over `sourceMiniBlocks` will "treat" the nil case

Reply:
right, but if they are not nil but empty, the `destMiniBlock` will be nil instead of empty.

Reply:
yes, but if you remove also L764, i think it will be fine; or maybe you can init like `make([]*hyperOutportBlocks.MiniBlock, len(sourceMiniBlocks)`

Reply:
if I init, I won't have parity between the two, since the initialized will never be nil, even though the source is nil.